### PR TITLE
FE-202: Intermediate CSS: Journey Into Mobile

### PR DESCRIPTION
**Pull request by nicoRDL**
## Description

- updated article pages to be responsive
- updated browse page to be responsive
- updated home page to be responsive

## Related Ticket

https://codeandtheory.atlassian.net/browse/FE-651

## Testing Instructions

1. Pull on branch Fe-202/FE-651
2. Test responsiveness of Article pages, browse page, and home page

## Risks
N/A

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -437,3 +422,395 @@ input {
...
+  }
+
+  .form-section {
+    grid-column: 2/-2;
+  }
+}
+
+/* DESKTOP BREAKPOINT */
+
+@media only screen and (min-width: 64rem) {
```
Comment (lycoangelo) [inline css/main.css]: Let's use `px` for media query. Browsers also uses pixel unit.

File: css/main.css [hunk 2]
```diff
@@ -437,3 +422,395 @@ input {
   text-transform: uppercase;
   margin: 1rem auto;
 }
+
+/* TABLET BREAKPOINT */
+
+@media screen and (min-width: 48rem) {
```
Comment (lycoangelo) [inline css/main.css]: Let's change unit to pixels.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

**Comment by nicoRDL** [comment]
@lycoangelo changed media query units to pixels 876b3f8f637eb9b8bddc646568f73877a4fda6c1

Also, added the rest of the responsive code for the browse page and home page in said commit for your evaluation.

File: css/home-page.css [hunk 3]
```diff
@@ -153,57 +100,215 @@
...
+  }
+  
+  .about-author-section img {
+      float: left;
+      max-width: 10rem;
+  }
+
+  .footer {
+      margin-top: 3rem;
+  }
```
Comment (lycoangelo) [inline css/home-page.css:312]: Indentation was inconsistent. Let's apply two spaces globally instead.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: css/home-page.css (hunk 3 above)
Comment (nicoRDL) [reply css/home-page.css:312]: b66e96c6d42ffa0fe112f188d123a36f4f6cc858 formatting fixed

@lycoangelo

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**